File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver){
        super(license, driver);
    }
    
    //Esto es polimorfismo
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    TypeCarAccepted = []
    SealtMaterial = []

    def __init__(self, license, driver, TypeCarAccepted, SealtMaterial):
        super().__init__(license, driver)

        self.TypeCarAccepted = TypeCarAccepted
        self.SealtMaterial = SealtMaterial